src/rules/spaceAfterTypeColon.js

```diff
@@ -1,4 +1,5 @@
 import _ from 'lodash';
+import {format} from 'util';
```

Comment:
Please remove `format` in favour of string concatenation (as it was).

Your are welcome to submit a separate PR promising to use `util.format`, though this addition is not required to solve this issue.